phoenicis-javafx/src/main/java/org/phoenicis/javafx/controller/apps/AppsController.java

```diff
@@ -64,4 +77,38 @@ private void showError(Exception e) {
             errorDialog.showAndWait();
         });
     }
+
+    /**
+     * sets the default category icons from the repository for the sidebar buttons
+     *
+     * @param categories categories in repository
+     */
+    private void setDefaultCategoryIcons(List<CategoryDTO> categories) {
+        Platform.runLater(() -> {
+            try {
+                StringBuilder cssBuilder = new StringBuilder();
+                for (CategoryDTO category : categories) {
+                    cssBuilder.append(
+                            "#" + ApplicationSidebarToggleGroupSkin.getToggleButtonId(category.getId()) + "{\n");
+                    URI categoryIcon = category.getIcon();
+                    if (categoryIcon == null) {
+                        cssBuilder
+                                .append("-fx-background-image: url('/org/phoenicis/javafx/views/common/phoenicis.png');\n");
+                    } else {
+                        cssBuilder.append("-fx-background-image: url('" + categoryIcon + "');\n");
+                    }
```

Comment:
The formatting looks really strange here. How about we change it to something like:

```java
final String categoryIconPath = Optional.ofNullable(category.getIcon())
    .map(categoryIcon -> categoryIcon.toString())
    .orElse("/org/phoenicis/javafx/views/common/phoenicis.png");

cssBuilder.append(String.format("-fx-background-image: url('%s');\n", categoryIconPath));
```

Reply:
I just copied the method from `MainController`.

Reply:
I know I just think it looks better this way and you are already touching the code as it is...